### PR TITLE
Downgrade nw-builder to version 3.8.3

### PR DESCRIPTION
**Pull request by McGiverGim**
Since version 3.8.4, nw-builder has a dependency with nw-install, that is not available at this moment. Downgrade to 3.8.3 since it is available again or we migrate to version 4.x.x.

### CODE REVIEW

**Comment by McGiverGim** [comment]
It seems Mac build is failing. I don't have a  Mac so I can't work on it. Some idea?

**Comment by McGiverGim** [comment]
Removing the nw.js cache, did the trick. The Mac build shows some ugly error, but maybe is a warning, it has finished, the artefact has been built but I can't test it.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by nerdCopter** [comment]
i see the OSX `node gyp` stuff:
` gyp ERR! not ok `
`node-pre-gyp ERR! not ok `.
I cannot test OSX either.

**Comment by nerdCopter** [comment]
just for reference/records - a more useful description of node-gyp than the npm description: https://stackoverflow.com/a/39741537

**Comment by haslinghuis** [comment]
yarn gulp release --osx64

```
Create release folder in /home/mark/dev/pr/configurator/betaflight-configurator/apps/betaflight-configurator/osx64
[19:15:36] Finished 'apps' after 3.73 min
[19:15:36] Starting 'cordova_dist_none'...
[19:15:36] Finished 'cordova_dist_none' after 237 μs
[19:15:36] Starting 'post_build_none'...
[19:15:36] Finished 'post_build_none' after 312 μs
[19:15:36] Starting '<anonymous>'...
[19:15:36] '<anonymous>' errored after 3.45 ms
[19:15:36] Error: Cannot find module 'appdmg'
Require stack:
- /home/mark/dev/pr/configurator/betaflight-configurator/gulp-appdmg.js
- /home/mark/dev/pr/configurator/betaflight-configurator/gulpfile.js
- /home/mark/dev/pr/configurator/betaflight-configurator/node_modules/gulp-cli/lib/versioned/^4.0.0/index.js
- /home/mark/dev/pr/configurator/betaflight-configurator/node_modules/gulp-cli/index.js
- /home/mark/dev/pr/configurator/betaflight-configurator/node_modules/gulp/bin/gulp.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/mark/dev/pr/configurator/betaflight-configurator/gulp-appdmg.js:2:16)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at release_osx64 (/home/mark/dev/pr/configurator/betaflight-configurator/gulpfile.js:836:20)
    at /home/mark/dev/pr/configurator/betaflight-configurator/gulpfile.js:913:20
    at bound (node:domain:421:15)
    at runBound (node:domain:432:12)
    at asyncRunner (/home/mark/dev/pr/configurator/betaflight-configurator/node_modules/async-done/index.js:55:18)
[19:15:36] 'release' errored after 4.2 min
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
mark@thinkpad:~/dev/pr/configurator/betaflight-configurator$
```

**Comment by McGiverGim** [comment]
Have you executed it under MacOS? The CI has finished, maybe does not work, but has finished.

**Comment by haslinghuis** [comment]
No MacOS for me.

**Comment by McGiverGim** [comment]
Then you can't build it, I think.

**Comment by McGiverGim** [comment]
What I suspect, seeing the link, is that some dependency fails to build under MacOs. The question is if that dependency is being used or not.

Now the nightlies are broken, so I think is better to merge this, that maybe has MacOs broken (or not) but the rest must work at least. It's the easiest way to see if we have complains about MacOs being broken too.

**Comment by haslinghuis** [comment]
Link to [OSX artifact](https://github.com/betaflight/betaflight-configurator/suites/13188362949/artifacts/717997713)

**Comment by nerdCopter** [comment]
https://github.com/betaflight/betaflight-configurator/suites/13188362949/artifacts/717997713

**Review by blckmn — APPROVED** [review]

**Comment by chmelevskij** [comment]
https://github.com/betaflight/betaflight-configurator/pull/3462#issuecomment-1570267108

☝️ seems to be working fine for me

**Pull request merged**
